assets/sass/_general.scss

```diff
@@ -88,6 +88,15 @@
   margin-bottom: govuk-spacing(4);
 }
 
+.black-background .govuk-hub-comment-form .govuk-fieldset__legend,
+.black-background .govuk-hub-comment-form .govuk-label {
+  color: govuk-colour('white');
+}
+
+.black-background #feedback-widget .govuk-link {
+  @extend .govuk-link--inverse;
```

Comment:
Does this need to be extended? can the govuk-link--inverse class not be added directly to the element. My understanding of @extend is that it takes styles for from example 'govuk-link--inverse' and adds additional styles in addition to the govuk-link--inverse styles?

Reply:
Because this only applies to the black-background version. The white background version should not have this class applied. We're conditionally applying(extending) the style for the black-background version.